ctlearn/default_models/single_cnn.py

```diff
@@ -45,15 +50,13 @@ def single_cnn_model(data, model_params):
             )(network_input_img)
 
         engine_output_cnn = engine_cnn(network_input_img, params=model_params, name=backbone_name)
-        engine_output_mlp = engine_mlp(network_input_param, params=model_params, name=backbone_name)
-
         output_cnn = tf.keras.layers.GlobalAveragePooling2D(
             name=backbone_name + "_global_avgpool"
         )(engine_output_cnn)
-        output_mlp = tf.keras.layers.Flatten()(engine_output_mlp)
-        concat = tf.keras.layers.Concatenate()([output_cnn, output_mlp])
-        
-        singlecnn_model = tf.keras.Model(inputs=[network_input_img, network_input_param], outputs = [concat], name=backbone_name)
-        
-
-    return singlecnn_model, [network_input_img, network_input_param]
+        if flag_prm == 1:
+            concat = tf.keras.layers.Concatenate()([output_cnn, output_mlp])
+            singlecnn_model = tf.keras.Model(inputs=[network_input_img, network_input_param], outputs = [concat], name=backbone_name)
+            return singlecnn_model, [network_input_img, network_input_param]
+        else:
+            singlecnn_model = tf.keras.Model(network_input_img, output_cnn, name=backbone_name)
+            return singlecnn_model, [network_input_img]
```

Comment:
It is better to only return once in a function. I'd suggest appending the model inputs/outputs depending on what is selected in the config. We can switch to slack for quicker interaction.

Reply:
Okay I will change this as well.

Reply:
done